spiketoolkit/sortingcomponents/detection.py

```diff
@@ -68,13 +75,16 @@ def detect_spikes(recording, channel_ids=None, detect_threshold=5, n_pad_ms=2, u
                                          for ch in channel_ids)
         for o in output:
             spike_times.append(o[0])
-            labels.append(o[1])
+            spike_amplitudes.append(o[1])
+            labels.append(o[2])
     else:
         for ch in channel_ids:
-            peak_times, label = _detect_and_align_peaks_single_channel(recording, ch, detect_threshold, detect_sign,
-                                                                       n_pad_samples, upsample, min_diff_samples,
-                                                                       align, start_frame, end_frame, verbose)
+            peak_times, peak_val, label = _detect_and_align_peaks_single_channel(recording, ch, detect_threshold,
+                                                                                 detect_sign, n_pad_samples, upsample,
+                                                                                 min_diff_samples, align, start_frame,
+                                                                                 end_frame, verbose)
```

Comment:
I don't anderstand the change here.

Reply:
I added `peak_val` as return. So detection now returns both spike times and an average amplitude